stable/selenium/templates/hub-service.yaml

```diff
@@ -16,6 +16,10 @@ spec:
   {{- if .Values.hub.serviceLoadBalancerIP }}
   loadBalancerIP: {{ .Values.hub.serviceLoadBalancerIP | quote }}
   {{- end }}
+  {{- if .Values.hub.loadBalancerSourceRanges }}
+  loadBalancerSourceRanges:
+  {{ toYaml .Values.hub.loadBalancerSourceRanges | indent 4 }}
```

Comment:
isn't the indent wrong?
Usually this would have no leading whitespace, and then indent 4 would be right - this code starts with two whitespaces, and would be indent 2.
Remove the leading whitespaces and it will be ok